src/engine/source/kvdb/include/kvdb/refCounter.hpp

```diff
@@ -0,0 +1,84 @@
+#ifndef _KVDB_REF_COUNTER_H
+#define _KVDB_REF_COUNTER_H
+
+#include <map>
+#include <string>
+#include <vector>
+
+namespace kvdbManager
+{
+
+/**
+ * @brief Basic Helper wrapping std::map<std::string, uint32_t> to count references.
+ *
+ */
+class RefCounter
```

Comment:
does it make sense to add UT to this class?

Reply:
This class is covered with the UTs that were put together in KVDBTest.